app/views/admin/docs/edit.html.erb

```diff
@@ -1,9 +1,9 @@
 <h2 class="form-subhead"><%= t(:edit, default:'Edit') %>: <%= @doc.title %></h2>
 <%= locale_select if AppSettings['i18n.available_locales'].count > 1 %>
-<%= bootstrap_form_for :doc, url: admin_category_doc_path(category_id: @category.id, id: @doc.id), :method => 'patch' do |f| %>
+<%= simple_form_for :doc, url: admin_category_doc_path(category_id: @category.id, id: @doc.id), :method => 'patch' do |f| %>
 
```

Comment:
I noticed this one is missing `validate: true`  Also, there is an exception on submitting invalid data:

```
undefined method `id' for nil:NilClass
Extracted source (around line #3):
         
<h2 class="form-subhead"><%= t(:edit, default:'Edit') %>: <%= @doc.title %></h2>
<%= locale_select if AppSettings['i18n.available_locales'].count > 1 %>
<%= simple_form_for :doc, url: admin_category_doc_path(category_id: @category.id, id: @doc.id), :method => 'patch' do |f| %>

  <% Globalize.with_locale(params['lang']) do %>
  <%= render 'shared/error_messages', object: f.object %>
```